rero/rero-ils
# UI: handle error

* NEW Adds route wildcard for Page Not Found (404)
* NEW Adds interceptor to catch http error

Signed-off-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

How to test:
- start angular server with `pipenv run npm run proxy`
- Test on `http://localhost:4200`
- Test url that does not exist. You are a "Page not found" error
- Test for interceptor: Logout the current Url. Check url `http://localhost:4200/records/item_types`
